tests/test_extensions.py

```diff
@@ -282,7 +282,7 @@ def setup(self):
             'first':
                 Dimension(MyTable.first),
             'firstanon':
-                Dimension(MyTable.first, anonymizer='{fake.name}'),
+                Dimension(MyTable.first, anonymizer='{fake:name}'),
```

Comment:
interesting. I was actually proposing keeping the `fake.name` syntax, but replacing `fake.name!foo...` with `fake.name:foo...`. but this works too :joy: